### ISSUE TITLE
[BUG] Inconsistent semantic of Ctrl-leftarrow and Ctrl-rightarrow in editor

### ISSUE THREAD

**Issue by fwjmath** [PremierLangage/platon-front#62]
In the editor, Ctrl-leftarrow works like in many graphical text editing interfaces, but not Ctrl-rightarrow, as it has been redefined.

**Reproduction**
1. Open a file with several words, locate to the middle of a word.
2. Type Ctrl-leftarrow, then Ctrl-rightarrow.

**Expected Behavior**
The cursor should move to the beginning of the word, then to the end of the word.

**Actual Behavior**
The cursor first moves to the beginning of the word, then the editing space is split into two, with the current file opened again in the second half.

**Possible solution**
Redefine the shortcut for Ctrl-rightarrow, or remove the shortcut if splitting is not a functionnality being frequently used.

**Comment by nthiery** [comment]
Yes, pleaaaaaase! Ctrl-arrow for moving around is hard coded in my fingers and I find the current behavior very annoying, especially since
there is no obvious way to undo the splitting.

**Comment by mciissee** [comment]
Sorry but this repository is linked to the next major version (1.0.0) of platform, the issue you are describing here is related to the editor of the current version. All the issues related to the current version (0.7.4) should be created on the repository https://github.com/PremierLangage/premierlangage.

In any case I will move this issue to the right repository now and close it once the issue will be resolved.
